app/views/discovered_hosts/_discovered_hosts_list.html.erb

```diff
@@ -1,8 +1,7 @@
-<%= javascript "host_checkbox" %>
```

Comment:
this is the actual fix, all the other changes are to avoid the deprecation warnings.